CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+### Changed
+
+### Removed
+
+### Fixed
+- Remove unused meeting session status codes from the meeting event guide and add more error messages
```

Comment:
Should we move this to "removed" section :) ?

Reply:
A good point - I'll update the description as I have more fixes.